lib/realtime/tenants/repo/migrations/20220312004840_update_apply_rls_function_to_apply_iso_8601.ex

```diff
@@ -1,8 +1,9 @@
-defmodule Realtime.Repo.Migrations.UpdateChangeTimestampToIso8601ZuluFormat do
+defmodule Realtime.Tenants.Repo.Migrations.UpdateApplyRlsFunctionToApplyIso8601 do
```

Comment:
add please `@moduledoc` tag to all new modules to keep in silence format checking

Reply:
done (e.g. https://github.com/supabase/realtime/pull/520/files#diff-be6c9758e10dc5f57ffc5c24343d4edaa874152b1fc0d63b137a4bac9d29bd71R2)